worker.go

```diff
@@ -333,6 +339,14 @@ func matchIngressesToLoadBalancers(loadBalancers []*loadBalancer, certs Certific
 		// limit is exeeded.
 		added := false
 		for _, lb := range loadBalancers {
+			// TODO(mlarsen): hack to phase out old load balancers
+			// which can't be updated to include type
+			// specification.
+			// Can be removed in a later version
```

Comment:
Should we document the upgrade procedure or do we want to create these if we tested it successfully in one of our production clusters?

Reply:
I would document the update procedure when we remove this comment at a later stage. Right now it works fine to upgrade (at least from my tests :))

The procedure would be to first update to the version which includes these changes and then later to a future version were this hack is removed.